.github/release_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+
+Git & Code
+- [ ] [GIT] Create branch release/[major].[minor].[patch] (freeze the code)
+- [ ] [DEV] Update version number [major].[minor].[patch]
+- [ ] [DIS] Update changelogs:
+     - [ ] [DEV] Update changelog https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md
+     - [ ] [GIT] Add the new release on GitHub https://github.com/owncloud/ios-app/releases
+- [ ] [QA] Design Test plan
+- [ ] [QA] Regression Test plan
+- [ ] [DOC] Update user manual with the new functionalities
+
+App Store
+- [ ] [DIS] Update screenshots if needed
+- [ ] [DIS] Upload the binary to the App Store
+- [ ] [DIS] App Store configuration: When release (manually, automatically, automatically not early than date)
+- [ ] [DIS] App Store configuration: How to releae (immediately to all the users o 7-day period phased release)
+- [ ] [DIS] App Store configuration: Reset iOS Summary Rating (Keep existing or reset the rating)
+- [ ] [DIS] Release to App Store
+- [ ] [DIS] Update changelogs
+
+Git
+- [ ] [GIT] Merge branch release/[major].[minor].[patch] in master
+- [ ] [GIT] Create tag and sign it "version_[major].[minor].[patch]"
```

Comment:
We're following the vanilla OneFlow concept (http://endoflineblog.com/oneflow-a-git-branching-model-and-workflow#finishing-a-release-branch), which names that tag just `[major].[minor].[patch]`

For better understanding, maybe change this to 

> Tag release commit as "[major].[minor].[patch]"